nautobot/extras/constants.py

```diff
@@ -1,7 +1,9 @@
 from django.db.models import Q
 
 # Approval Workflow models
-APPROVAL_WORKFLOW_MODELS = Q(app_label="extras", model__in=("scheduledjob",))
+APPROVAL_WORKFLOW_MODELS = Q(app_label="extras", model__in=("scheduledjob",)) | Q(
```

Comment:
Hmm, we're going to need a mechanism for apps to add models to this list in the future without needing a core change every time. Can we refactor this implementation to use `FeatureQuery` and `populate_model_features_registry` instead of a constant?